relations/src/r1cs/constraint_system.rs

```diff
@@ -240,12 +240,12 @@ impl<F: Field> ConstraintSystem<F> {
 
     /// Count the number of times a given LC is used within another LC
```

Comment:
```suggestion
    /// Count the number of times each LC is used within other LCs in the constraint system
```

Reply:
I know this wasn't part of this PR, but since this was being edited figured might as well improve the comments